source/Analysis/include/AnalyzerGeometry.h

```diff
@@ -37,6 +37,16 @@ class TProfile;
 class Ring;
 class RootWTable;
 
+
+// Formatting parameters
+static const int coordPrecision = 3;
+static const int zOverlapPrecision = 5;
+static const int anglePrecision = 1;
+// Some strings for the html formatting
+static const std::string subStart   = "<sub>";      // These only should be needed
+static const std::string subEnd     = "</sub>";
+
```

Comment:
Gabi, the global constants are not needed in various analyzers. They should be defined just as local member constants. In addition, subStart, subEnd is already defined in global_constants.h file ... Can you ,please, change it?